support-services/src/main/scala/com/gu/aws/AwsCloudWatchMetricPut.scala

```diff
@@ -46,17 +45,26 @@ object AwsCloudwatchMetricPut {
     metricDatum1.setValue(1.00)
     metricDatum1.setUnit(StandardUnit.Count)
     putMetricDataRequest.getMetricData.add(metricDatum1)
-    Try(s3Client.putMetricData(putMetricDataRequest)).map(_ => ())
+    Try(client.putMetricData(putMetricDataRequest)).map(_ => Unit)
```

Comment:
Old PR I know, but I don't think this change is correct.
`Unit` is the type, `()` is the value.  `Unit` is a companion object of type `Unit.type` which is not in itself of type `Unit`.
This does happen to pass the type checker, because it will discard a non Unit value in Unit context, and it does work at run time because the value is discarded, but it's technically wrong 😎